src/devices/ServoMotor/ServoMotor.cs

```diff
@@ -0,0 +1,133 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Pwm;
+
+namespace Iot.Device.ServoMotor
+{
+    /// <summary>
+    /// Represents a rotary actuator or linear actuator that allows for precise control of angular or linear position.
+    /// </summary>
+    public class ServoMotor : IDisposable
+    {
+        private PwmChannel _pwmChannel;
+        private readonly int _maximumAngle;
```

Comment:
minimum is zero? is this in degrees, radians, grad, some other unit? Why is this an int and not say double? Some comment would be useful here

Reply:
The angle is usually represented in degrees, so maybe I don't have the correct naming for variables.  Maybe should change to `angleDegrees` like we did with `pulseWidthMicroseconds`.  I think using degrees (versus 0 - 1 values) is a little better as it gives a better picture of how far you're actually changing the position.

Reply:
> The angle is usually represented in degrees

@shaggygi I tend to disagree - I don't think we ever used degrees when I was studying - it was always radians. Same in graphics - only radians (except maybe UI). The reason is that it combines really well with the rest of the math: i.e you can pass it directly to Math.Sin or calculate physical distance traversed without doing extra conversions.

I guess I don't care that much either way but for public APIs units should be very clear and ideally added in the name (i.e. MoveByDegrees). Other option we can add another struct in the `Units` project then problem will be solved.

Reply:
either way - if unit is not clear it should be in the name or the comment

Reply:
maybe we just name parameter degrees and _maximumDegrees